src/QsFmt/App/Program.fsi

```diff
@@ -7,3 +7,4 @@ module Microsoft.Quantum.QsFmt.App.Program
 /// Runs the Q# formatter.
 [<CompiledName "Main">]
 val main: string [] -> int
+
```

Comment:
Huh... this extra newline snuck back in again. I wonder if it's a formatting tool making this change?

Reply:
Weirdly persistent little new line. I've removed it again.